src/react/components/Navigation.js

```diff
@@ -0,0 +1,56 @@
+import React from 'react';
+import { useParams, useNavigate, useLocation } from 'react-router-dom';
+
+import home from '../../assets/images/home.png';
+import back from '../../assets/images/back.png';
+import settings from '../../assets/images/settings.png';
+import mic from '../../assets/images/mic.png';
+
+const Navigation = () => {
+  const navigate = useNavigate();
+  const params = useParams();
+  const location = useLocation();
+
+  const handleBack = () => {
+    if (location.pathname === '/') return;
+    navigate(-1);
+  };
+
+  return (
+    <div className="nav" data-testid="nav-a">
+      <div className="back-button">
+        <div
+          className="back"
+        >
+          <button
+            onClick={handleBack}
+            type="button"
+            className="backb"
+          >
+            {location.pathname === '/' ? <img src={home} alt="" className="imgw" /> : <img src={back} alt="" className="imgw" />}
+          </button>
+        </div>
+        <div className="year">{Object.keys(params).length === 1 ? '2020-03-22' : ''}</div>
```

Comment:
- [ ] Kindly add some functionality for user's where we can filter data, this can be done either with a search input field or other means. I have said so because it's one of the instructions for the project. In the project instructions, under interactions then the first point.
```
When the page loads, the webapp shows the list of categories that could be filtered out by some parameter, for example by the category name.
```

Reply:
Thanks @elisha2kyakpo1.